modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -5,7 +5,13 @@
 [id="nw-ovn-kubernetes-live-migration-about_{context}"]
 = Limited live migration to the OVN-Kubernetes network plugin overview
 
-The limited live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}.
+The limited live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. For {product-title} version 4.15, it is available for versions 4.15.31 and later.
```

Comment:
One suggestion and it's up to you, but I feel like we often drop "version" here:
```suggestion
The limited live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. For {product-title} 4.15, it is available for versions 4.15.31 and later.
```

Not sure if there's anything in the style guides aside from examples to version references, but it might be more consistent with other places in our docs.

Reply:
Huh. I don't know why I thought we were supposed to use version. Maybe it's for a different context. Fixed anyways.